src/parser/data-purge-module.ts

```diff
@@ -26,6 +26,8 @@ export class DataPurgeModule implements Module {
           .replace(/^>.*$/gm, "")
           // Remove commands such as /start
           .replace(/^\/.+/g, "")
+          // Remove HTML comments
+          .replace(/<!--[\s\S]*?-->/g, "")
```

Comment:
If we use a virtual dom creator like jsdom or mdast, we should be able to query element.textContent and it should handle this and other situations in a robust manner.

Reply:
Body comes in a text form. Then it gets transformed into MD -> HTML. So it is rendered as a text form. So typically when fetched from GitHub the body looks like
`Resolves #23 <!-- comment -->` so I don't see a way to skip it other than removing it.

Reply:
HTML comments shouldn't be included in element.textContent is my point

Reply:
Yes, but there is no way to know it is a comment before converting it to HTML, since the MD renderer is ran first
https://github.com/gentlementlegen/conversation-rewards/blob/cf5ecb6a9f1bb551fa01c866d6f8ccaefa7e1804/src/parser/formatting-evaluator-module.ts#L112

(it is actually the same for v1) so it is first converted to a `p` that contains the comment.